src/utils/defaultReqsConfig.ts

```diff
@@ -0,0 +1,11 @@
+import { RequestConfiguration } from './cancelRequests';
```

Comment:
`RequestConfiguration` and  `getAxiosReqParams`  should probably be moved from cancelRequests to some other location at some point as they are no longer used only for canceling requests .fftr

Reply:
I agree - but I'm reluctant to do this change in this MR. I wanted the change to be small (because it was a hotfix), and now that it is done and tested, I'd prefer if the refactoring was done sometime in the future.